### PR TITLE
[IMP] Change validating sheets warning

### PR DESCRIPTION
**Pull request by MatteoLdx**
Don't display the warning when validating attendance sheets if there's no absents

### CODE REVIEW

File: beesdoo_shift_attendance/models/attendance_sheet.py [hunk 1]
```diff
@@ -227,6 +227,12 @@ class AttendanceSheet(models.Model):
         )
     ]
 
+    # True if there are one or more absents
+    absents = fields.Boolean(
```
Comment (robinkeunen) [inline beesdoo_shift_attendance/models/attendance_sheet.py]: Rather `missing_worker` (+ compute function)

Comment (robinkeunen) [reply beesdoo_shift_attendance/models/attendance_sheet.py]: `has_missing_worker`

File: beesdoo_shift_attendance/models/attendance_sheet.py [hunk 2]
```diff
@@ -284,6 +290,17 @@ def _compute_is_annotated(self):
...
+    def _compute_absents(self):
+        for rec in self:
+            rec.absents = False
+            for shift in rec.expected_shift_ids:
+                if shift.state != "done":
+                    rec.absents = True
+            for shift in rec.added_shift_ids:
+                if shift.state != "done":
+                    rec.absents = True
+
```
Comment (robinkeunen) [inline beesdoo_shift_attendance/models/attendance_sheet.py]:
```suggestion
    @api.depends("expected_shift_ids", "added_shift_ids")
    def _compute_absents(self):
        for rec in self:
            if any((s.state != 'done' for s in rec.expected_shift_ids)):
                rec.absents = True
                continue 
            if any((s.state != 'done' for s in rec.added_shift_ids)):
                rec.absents = True
                continue
```

**Review by robinkeunen — DISMISSED** [review]

**Review by remytms — COMMENTED** [review]
@MatteoLdx LGTM, but squash your commits. :)

**Review by remytms — APPROVED** [review]
OK

**Pull request merged**